spec/features/memberships/sac_terminate_membership_wizard_spec.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+require "spec_helper"
+
+describe "terminate sac membership wizard", js: true do
+  let(:person) { people(:mitglied) }
+  let(:role) { person.roles.first }
+  let(:operator) { person }
+  let(:group) { groups(:bluemlisalp_mitglieder) }
+  let(:termination_reason) { termination_reasons(:deceased) }
+
+  before do
+    sign_in(operator)
+  end
+
+  context "as SAC Mitarbeiter" do
+    let(:operator) { people(:admin) }
+
+    xit "can execute wizard with immediate deletion" do
```

Comment:
Why is this skipped?
```suggestion
    it "can execute wizard with immediate deletion" do
```

Reply:
todo spec still does not work (should be a new issue0